src/python/bot/tasks/fuzz_task.py

```diff
@@ -726,6 +728,8 @@ def run_fuzzer(fuzzer, fuzzer_directory, testcase_directory, data_directory,
   testcase_manager.remove_testcases_from_directories(testcase_directories)
 
   # Set an environment variable for fuzzer name.
+  # TODO(ochang): Investigate removing this. Only users appear to be chromebot
```

Comment:
chromebot is no longer running, so it can be removed. only webbot. but yes fuzzer_logs is an active user.

Reply:
Ack. will be removing this in a future CL.